MLIR.cmake

```diff
@@ -141,6 +141,11 @@ function(add_onnx_mlir_library name)
     target_include_directories(${name} ${ARG_INCLUDE_DIRS})
   endif()
 
+  if (ARG_ACCEL_INCLUDE_DIRS)
```

Comment:
So now these libraries will be added to both global properties - is that the goal?

I think it would be better to make the function optionally have a global property to add to and just pass in a different property for both usages (or default to the current one).

Reply:
With your latest change, the DLC_LIBS are no longer being set, but all the accelerator libs are added to the OMLibs. Is that intentional?